mysql-test/suite/versioning/t/trx_id.test

```diff
@@ -411,3 +411,154 @@ DROP TABLE tts;
 DROP TABLE ttx;
 DROP FUNCTION fts;
 DROP FUNCTION ftx;
+
+--echo #
+--echo # MDEV-16330 Allow instant change of WITH SYSTEM VERSIONING column attribute
+--echo #
+
+SET @@SYSTEM_VERSIONING_ALTER_HISTORY=KEEP;
+CREATE TABLE t (
+  a INT,
+  b INT,
+  row_start BIGINT UNSIGNED AS ROW START INVISIBLE,
+  row_end BIGINT UNSIGNED AS ROW END INVISIBLE,
+  PERIOD FOR SYSTEM_TIME(row_start, row_end)
+) WITH SYSTEM VERSIONING ENGINE=INNODB;
+
+INSERT INTO t VALUES (1,1);
+
+--echo without table rebuild
```

Comment:
Start this kind of `--echo` output with `#`, so that the comment will better be distinguished from SQL statements and their output in the .result file.